clients/src/main/resources/common/message/EnvelopeRequest.json

```diff
@@ -23,7 +23,7 @@
   "fields": [
     { "name": "RequestData", "type": "bytes", "versions": "0+", "zeroCopy": true,
       "about": "The embedded request header and data."},
-    { "name": "RequestPrincipal", "type": "bytes", "versions": "0+", "zeroCopy": true, "nullableVersions": "0+",
+    { "name": "RequestPrincipal", "type": "bytes", "versions": "0+", "nullableVersions": "0+",
```

Comment:
Is ```nullableVersions`` required? It seems there is no null handle/check in production for this field.

Reply:
I agree it should not be nullable. Will fix it.